js/assessments/paragraphTooLongAssessment.js

```diff
@@ -97,9 +102,10 @@ var paragraphLengthMarker = function( paper, researcher ) {
 	var paragraphsLength = researcher.getResearch( "getParagraphLength" );
 	var tooLongParagraphs = getTooLongParagraphs( paragraphsLength );
 	return map( tooLongParagraphs, function( paragraph ) {
-		var marked = marker( paragraph.paragraph );
+		var paragraphText = paragraph.paragraph.replace( blockElementStartRegex, "" ).replace( blockElementEndRegex, "" );
```

Comment:
Also consider changing the replacing of block element starts / ends to be done via a function instead of this. The stripping could potentially be useful in other areas and has more to do with string manipulation than anything else. You could add this to `stripHTMLTags`.